ttnn/cpp/ttnn/operations/reduction/prod/prod.cpp

```diff
@@ -101,14 +106,15 @@ Tensor ProdOperation::invoke(
     // Permute and unpad result for dim 2,3
     auto step = ttnn::SmallVector<uint32_t>({1, 1, 1, 1});
     if (dim == 0 || dim == 1 || dim == -4 || dim == -3) {
```

Comment:
Just remove this if clause, then the rest becomes
```
if (dim == 2 || dim == -2) {
...
} else if (dim == 3 || dim == -1) {
...
```

You could change the previous comment to something like:
```
// Permute and unpad result for dim 2,3. Don't need to process dim 0,1.
```

Reply:
done